wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/CommitTest.java

```diff
@@ -111,6 +111,22 @@ public void testAmendCommit(GitConnectionFactory connectionFactory) throws GitEx
         assertEquals(beforeCount, afterCount);
     }
 
+    @Test(dataProvider = "GitConnectionFactory", dataProviderClass = org.eclipse.che.git.impl.GitConnectionFactoryProvider.class)
+    public void testChangeMessageOfLastCommit(GitConnectionFactory connectionFactory) throws GitException, IOException {
+        //given
+        GitConnection connection = connectToGitRepositoryWithContent(connectionFactory, repository);
+        int commitsBefore = connection.log(newDto(LogRequest.class)).getCommits().size();
+
+        //when
+        CommitRequest commitRequest = newDto(CommitRequest.class).withMessage("Changed message").withAmend(true);
+        connection.commit(commitRequest);
+
+        //then
+        int commitsAfter = connection.log(newDto(LogRequest.class)).getCommits().size();
+        assertEquals(commitsBefore, commitsAfter);
```

Comment:
Looks like you check equals of commits before and after your commit. But in fact you compare only commits number. As for me it would be better to rename this variable to make this aspect clear